clarify which tau in Praos reference

nfrisby opened the pull request:
There are at least two taus in the Praos paper: one is a parameter of the Chain
Growth property and the other is a parameter of the Resettable Leaky Beacon
Functionality. The SlotsPrior global constant refers to the RLB Functionality
parameter.

@JaredCorduan @uroboros, it is the RLB tau, right? Thanks!

nfrisby commented:
@JaredCorduan @uroboros FYI, I haven't rerendered locally, so this text might be too wide for the table column. Do you know the nix invocation to build it locally?

If we need to shorten the column, then "tau of F^{\tau,r}_{RLB}" might suffice.

uroboros commented:
@nfrisby you can use `nix-shell --pure --run make` to build the latex docs locally (there is more detail in the cardano-ledger-specs readme also)

uroboros commented on shelley/chain-and-ledger/formal-spec/protocol-parameters.tex:
```diff
@@ -166,7 +166,7 @@ \section{Protocol Parameters}
       \SlotsPerEpoch & \N & \text{slots per epoch} \\
       \SlotsPerKESPeriod & \N & \text{slots per KES period} \\
-      \SlotsPrior & \Duration & \tau\text{ in \cite{ouroboros_praos}}\\
+      \SlotsPrior & \Duration & \text{nonce leakage paramter }\tau\text{ in \cite{ouroboros_praos}}\\
```
"parameter"

uroboros commented:
> There are at least two taus in the Praos paper: one is a parameter of the Chain
> Growth property and the other is a parameter of the Resettable Leaky Beacon
> Functionality. The SlotsPrior global constant refers to the RLB Functionality
> parameter.
> 
> @JaredCorduan @uroboros, it is the RLB tau, right? Thanks!

@nfrisby yes I think you're right

nfrisby commented:
@uroboros I fixed the typo you spotted and updated several of the `default.nix` files as we discussed on Slack (see new commit). The `nix-shell --pure --run make` command now succeeds in all those files' directories.

I did not update these `default.nix` files, since I'm assuming the `executable-spec` infrastructure is correct (b/c I assume the CI exercises it).

```
$ head -n1000 $(git grep -l -F '../default.nix')
==> byron/chain/executable-spec/default.nix <==
let
  pkgs = import ../../../default.nix {};
in
  pkgs.libs.byron-spec-chain

==> byron/ledger/executable-spec/default.nix <==
let
  pkgs = import ../../../default.nix {};
in
  pkgs.libs.byron-spec-ledger

==> semantics/executable-spec/default.nix <==
let
  pkgs = import ../../../default.nix {};
in
  pkgs.libs.small-steps
```

nc6 commented:
Actually, those files are completely out of date, and should be removed. But that's independent of this PR, which looks good!

JaredCorduan commented on shelley/chain-and-ledger/formal-spec/protocol-parameters.tex:
```diff
@@ -166,7 +166,7 @@ \section{Protocol Parameters}
       \SlotsPerEpoch & \N & \text{slots per epoch} \\
       \SlotsPerKESPeriod & \N & \text{slots per KES period} \\
-      \SlotsPrior & \Duration & \tau\text{ in \cite{ouroboros_praos}}\\
+      \SlotsPrior & \Duration & \text{nonce leakage parameter }\tau\text{ in \cite{ouroboros_praos}}\\
```
thank you for clearing this us @nfrisby ! I had not realized that there were two `\tau`s, and "nonce leakage parameter" is better anyway.
